### PR TITLE
Adds a vue component to display tool errors in the admin panel

### PR DESCRIPTION
**Pull request by guerler**

### CODE REVIEW

File: lib/galaxy/webapps/galaxy/api/tools.py [hunk 1]
```diff
@@ -404,6 +405,15 @@ def fetch(self, trans, payload, **kwd):
         create_payload.update(files_payload)
         return self._create(trans, create_payload, **kwd)
 
+    @expose_api
+    @web.require_admin
+    def error_stack(self, trans, **kwd):
+        """
+        POST /api/tools/error_stack
```
Comment (dannon) [inline lib/galaxy/webapps/galaxy/api/tools.py]: Is this actually a post?  Seems like it should be a GET.

Comment (guerler) [reply lib/galaxy/webapps/galaxy/api/tools.py]: Yep, thats supposed to be a GET. Thanks.

Comment (dannon) [reply lib/galaxy/webapps/galaxy/api/tools.py]: Cool, thought so!

**Comment by jmchilton** [comment]
screen shot 2019-01-02 at 10 03 16 am

Works for me!

**Pull request merged**